internal/olm/operator/manager.go

```diff
@@ -53,6 +53,7 @@ func init() {
 type operatorManager struct {
 	client        *olmresourceclient.Client
 	version       string
+	olmNamespace  string
```

Comment:
Group this with `client` (move it up one position).

Reply:
fixed